perun-core/src/main/java/cz/metacentrum/perun/core/api/ConsentsManager.java

```diff
@@ -49,6 +50,19 @@ public interface ConsentsManager {
 	 */
 	List<Consent> getConsentsForConsentHub(PerunSession sess, int id) throws PrivilegeException, ConsentHubNotExistsException;
 
+	/**
+	 * Get all consents for ConsentHub corresponding to the facility of the chosen resource.
```

Comment:
also edit the docs. (we want to return only the consents of the members of that resource, not all consents for the facility)